php/libraries/Breadcrumb.class.inc

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * This file defines a Breadcrumb element.
+ *
+ * PHP Version 7
+ *
+ * @category Breadcrumbs
+ * @package  User_Interface
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS;
+
+/**
+ * A Breadcrumb consist of a label and an hyperlink.
+ *
+ * PHP Version 7
+ *
+ * @category Breadcrumbs
+ * @package  User_Interface
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Breadcrumb
+{
+    /**
+     * The label to display
+     *
+     * @var string
+     */
+    protected $label;
+
+    /**
+     * The hyperlink for that breadcrumb
+     *
+     * @var string
+     */
+    protected $link;
+
+    /**
+     * The contructor
+     *
+     * @param string $label The label to display
+     * @param string $link  The hyperlink for that breadcrumb
+     */
+    public function __construct(string $label = '', string $link = '')
+    {
+        $this->label = $label;
+        $this->link  = $link;
+    }
+
+    /**
+     * Returns a string representation of that object.
```

Comment:
Should specify what that string representation is and how it's different than the PHP default.

(Seems to be different in that it's serializing to JSON, which is fine, but should be documented.)